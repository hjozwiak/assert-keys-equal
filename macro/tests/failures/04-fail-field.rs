#[macro_use]
extern crate assert_keys_equal_macro;
use solana_program::pubkey::Pubkey;
struct Tester {
    key: Pubkey,
}

fn main() {
    let test = Tester {
        key: Pubkey::new_unique(),
    };
    assert_pubkeys_equal!(
        "I should be a field.",
        "my1111111111111111111111111111111111111111111111111111"
    );
}
