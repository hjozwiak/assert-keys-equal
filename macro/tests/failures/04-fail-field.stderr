error: expected struct field access
  --> $DIR/04-fail-field.rs:13:9
   |
13 |         "I should be a field.",
   |         ^^^^^^^^^^^^^^^^^^^^^^
